salt/metalk8s/saltmaster/files/minion_99-metalk8s.conf.j2

```diff
@@ -0,0 +1,5 @@
+master: localhost
+
+# use new module.run format
+use_superseded:
```

Comment:
(Rant) This 'feature' is so confusingly named (https://github.com/saltstack/salt/issues/42272) and lacks any proper documentation (https://docs.saltstack.com/en/latest/ref/states/all/salt.states.module.html confuses me more than providing any clarification).

Reply:
@sayf-eddine-scality is this solved?

Reply:
@TeddyAndrieux  what do you think